ws.py

```diff
@@ -35,6 +35,11 @@
 from tld.utils import update_tld_names, TldIOError
 from helpers import log
 
+import os.path
+
+if not os.path.islink(".git/hooks/pre-commit"):
+    os.symlink("hooks/pre-commit", ".git/hooks/pre-commit")
```

Comment:
What about copying `hooks/*` into `.git/hooks/` on each launch? This keeps the scripts up-to-date without the issues surrounding symlinks.